pydantic/fields.py

```diff
@@ -111,6 +135,30 @@ def from_field(cls, default: Any = Undefined, **kwargs: Any) -> FieldInfo:
         >>> class MyModel(pydantic.BaseModel):
         >>>     foo: int = pydantic.Field(4, ...)  # <-- like this
         """
+        """
+        Create a new `FieldInfo` object with the `Field` function.
+
+        Args:
+            cls (type): The class of the object to create.
```

Comment:
I think we don't need to document the `cls` arg in class methods like we don't document `self` in instance methods